MapboxMobileEvents/MMEMetricsManager.m

```diff
@@ -126,9 +126,11 @@ - (void)updateMetricsFromEventQueue:(NSArray *)eventQueue {
         self.metrics.eventCountTotal += (int)eventQueue.count;
         
         for (MMEEvent *event in eventQueue) {
-            NSNumber *eventCount = [self.metrics.eventCountPerType objectForKey:event.name];
-            eventCount = [NSNumber numberWithInteger:[eventCount integerValue] + 1];
-            [self.metrics.eventCountPerType setObject:eventCount forKey:event.name];
+            if (event.name) {
```

Comment:
An unnamed event at this point in execution is a reportable error, please add an else block with an error report and a new code for tracking

Reply:
I was considering removing this check but it's probably good to keep in place. Error report added. Thanks